config/locales/zh-CN.yml

```
@@ -482,7 +488,7 @@ zh-CN:
       released_with_mfa:
       links:
         badge: 徽章
-        bugs: Bug 追踪
+        bugs: Bug
```

For consistency and translation completness
```suggestion
        bugs: Bug 追踪
```